components/tokens/custom-spectrum/custom-light-vars.css

```
@@ -19,5 +19,14 @@ governing permissions and limitations under the License.
   --spectrum-menu-item-background-color-key-focus: var(--spectrum-transparent-black-200);
 
   /* Drop Zone background color rgb */
-  --spectrum-drop-zone-background-color-rgb: var( --spectrum-blue-800-rgb); /* var(--spectrum-accent-color-800);*/
+  --spectrum-drop-zone-background-color-rgb: var(--spectrum-blue-800-rgb); /* var(--spectrum-accent-color-800);*/
+
+  --spectrum-calendar-day-background-color-selected: rgba(2,101,220,0.1);
```

Are we able to use this instead?

```suggestion
  --spectrum-calendar-day-background-color-selected: rgba(var(--spectrum-blue-900-rgb), 0.1);
```